extension/src/vscode/title.ts

```diff
@@ -1,6 +1,7 @@
 export enum Title {
   CHOOSE_RESOURCES = 'Choose Resources to Add to the Dataset',
   ENTER_BRANCH_NAME = 'Enter a Name for the New Branch',
+  ENTER_RENAME_NAME = 'Enter a Name for the New Experiment',
```

Comment:
[Q] Should this be 'Enter a New Name for the Experiment'?

Reply:
Yes, that makes more sense

Reply:
probably a good idea to update `ENTER_RENAME_NAME` as well